helm/_shared/named_templates/_postgres.tpl

```diff
@@ -1,16 +1,12 @@
-{{- define "vro.postgresUrl" -}}
-{{- printf "jdbc:postgresql://%s-postgres:%s/%s"
-  .Values.global.hostnamePrefix
-  (toString .Values.global.service.db.targetPort)
-  .Values.global.service.db.databaseName }}
-{{- end }}
-
 {{/*
   For clients to connect to DB
 */}}
 {{- define "vro.dbClient.envVars" -}}
 - name: POSTGRES_URL
-  value: {{ include "vro.postgresUrl" . }}
+  valueFrom:
+    secretKeyRef:
+      name: rds-db
+      key: DB_URL
```

Comment:
Where is `DB_URL` env var injected? Is this a static url added to Github actions Env?

Reply:
This is a K8s secret